corebuild/integration/ILLink.Tasks/ILLink.Tasks.targets

```diff
@@ -166,22 +170,41 @@
                            Condition="Exists('%(Identity)') And '$(_DebugSymbolsProduced)' == 'true' " />
     </ItemGroup>
   </Target>
-  
+
+  <UsingTask TaskName="SetAssemblyActions" AssemblyFile="$(LinkTaskDllPath)" />
+  <Target Name="_SetAssemblyActions"
+          DependsOnTargets="_ComputeManagedAssembliesToLink">
+      <SetAssemblyActions AssemblyPaths="@(_ManagedAssembliesToLink)"
+                          ApplicationAssemblyNames="@(_ApplicationAssembliesToLink)"
+                          PlatformAssemblyNames="@(_PlatformAssembliesToLink)"
+                          UsedApplicationAssemblyAction="$(UsedApplicationAssemblyAction)"
+                          UnusedApplicationAssemblyAction="$(UnusedApplicationAssemblyAction)"
+                          UsedPlatformAssemblyAction="$(UsedPlatformAssemblyAction)"
+                          UnusedPlatformAssemblyAction="$(UnusedPlatformAssemblyAction)">
+        <Output TaskParameter="AssemblyPathsWithActions" ItemName="_ManagedAssembliesToLinkWithActions" />
+      </SetAssemblyActions>
+
+      <ItemGroup>
+          <_ManagedAssembliesToLink Remove="@(_ManagedAssembliesToLink)"/>
+          <_ManagedAssembliesToLink Include ="@(_ManagedAssembliesToLinkWithActions)"/>
```

Comment:
Extra space after `Include`. Please also put spaces before the closing `/>`.

Reply:
Done.